girder/__init__.py

```diff
@@ -35,6 +35,9 @@
 __license__ = 'Apache 2.0'
 
 
+Quiet = False
```

Comment:
It is a symbol within the module that is used to stop logprint from printing.  It could be manipulated other code, but is more likely just set by the configuration.  We could read it from the config whenever logprint is called, but that seems expensive.

I capitalized it because apparently my ingrained coding habits are module-globals are capitalized.  I'd be happy to change it.